specification/deviceprovisioningservices/data-plane/Microsoft.Devices/preview/2021-11-01-preview/device.json

```diff
@@ -0,0 +1,1044 @@
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CertificateIssuanceResponse"
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/ProvisioningServiceErrorDetails"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Returns the issued certificate": {
+            "$ref": "./examples/device/CertificateIssuance/getIssuedCertificate.json"
+          }
+        }
+      }
+    },
+    "/{idScope}/devices/{registrationId}/operations/{operationId}": {
+      "get": {
+        "tags": [
+          "GET"
+        ],
+        "summary": "Obtains the device operation status.",
+        "operationId": "Device_OperationStatusLookup",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "idScope",
+            "in": "path",
+            "description": "The service assigned ID scope.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "registrationId",
+            "in": "path",
+            "description": "The device registration ID. A case-insensitive string (up to 128 characters long) of alphanumeric characters plus certain special characters : . _ -. No special characters allowed at start or end.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "operationId",
+            "in": "path",
+            "description": "Operation ID. A case-insensitive string (up to 128 characters long) of alphanumeric characters plus certain special characters : . _ -. No special characters allowed at start or end.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DeviceOperationStatusResponse"
+            },
+            "headers": {
+              "retry-after": {
+                "description": "The retry interval in seconds.",
+                "type": "integer",
+                "format": "int32"
+              },
+              "location": {
+                "description": "The location of the resource.",
+                "type": "string"
+              }
+            }
+          },
+          "200": {
```

Comment:
Should just consistently return 202 or 200 (likely 202 since it could be an LRO), but not both. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations-with-status-monitor